src/pages/filter.css

```diff
@@ -0,0 +1,101 @@
+.filter-container{
+    width:15%;
+    background-color: aliceblue;
+    display: flex;
+    flex-direction: column;
+    gap:1rem;
+    padding-left:1rem;
+    padding-right:1rem;
+    padding-bottom:1rem;
+    border-radius:15px;
+}
+:root{
```

Comment:
suggestion: color variables could be defined in a common css file like the app.css